src/service/poc_mobile.proto

```diff
@@ -438,6 +438,90 @@ message radio_reward {
   repeated boosted_hex boosted_hexes = 10;
 }
 
+enum boosted_hex_status {
+  boosted_hex_status_eligible = 0;
+  boosted_hex_status_location_score_below_threshold = 1;
+  boosted_hex_status_radio_threshold_not_met = 2;
+  boosted_hex_status_service_provider_ban = 3;
+}
+
+message radio_reward_v2 {
+
+  message location_trust_score {
+    uint64 meters_to_asserted = 1;
+    // value is 0.0 to 1.0 multiplied by 1000
+    uint32 trust_score = 2;
+  }
+
+  message covered_hex {
+    uint64 location = 1;
+    // modeled coverage points * rank_multiplier * assignment_multiplier
+    uint64 base_coverage_points = 2;
+    // base_coverage_points * (boosted_multiplier - 1)
+    uint64 boosted_coverage_points = 3;
```

Comment:
Does it mean that earned `coverage_points` for particular hex is `base_coverage_points` + `boosted_coverage_points`?

Reply:
yes